.travis.yml

```diff
@@ -28,6 +33,33 @@ script:
   - pushd cherry_picker
   - pytest cherry_picker/test.py -v
   - popd
+- &deploy-base
+  stage: Publish dists to PYPI
+  python: "3.6-dev"
+  script:
+  - flit build
+  # Add an empty setup.py stub, because pypi provider always calls it
+  - touch setup.py
+  deploy:
+    provider: pypi
+    # `skip-cleanup: true` is required to preserve binary wheel and sdist,
+    # built by during `install` step above.
+    skip-cleanup: true
+    # `skip-existing: true` is required to skip uploading dists, already
+    # present in PYPI instead of failing the whole process.
+    # This happenes when other CI (AppVeyor etc.) has already uploaded
+    # the very same dist (usually sdist).
+    skip-existing: true
+    user: Mariatta  # TODO: consider having a separate "uploader user"
+                    # with reduced priviliges for security reasons
+    password:
+      # Encrypt with `travis encrypt -r python/core-workflow --org` while using travis-ci.org;
+      # change to `travis encrypt -r python/core-workflow --api-endpoint 'https://api.travis-ci.com/'`
+      # upon switch to __free__ travis-ci.com:
+      secure: PLACE_YOUR_ENCRYPTED_PASSWORD_HERE
```

Comment:
@Mariatta feel free to update the PR branch with a secret generated for your account as per instructions in comments above.